lib/srv/regular/sshserver.go

```diff
@@ -308,6 +311,12 @@ func (s *Server) GetHostUsers() srv.HostUsers {
 	return s.users
 }
 
+// GetHostSudoers returns the HostSudoers instance being used to manage
+// sudoers file provisioning
+func (s *Server) GetHostSudoers() (srv.HostSudoers, error) {
+	return s.sudoers, nil
```

Comment:
The whole point of the fallible API was to avoid segfaults due to accidentally not checking the return value for `nil`, but `s.sudoers` as returned by `srv.NewHostSudoers` will happily be `nil` on darwin.

This needs something like:
```suggestion
	if s.sudoers == nil {
		return nil, trace.NotImplemented("sudoers functionality is not available on this platform")
	}
	return s.sudoers, nil
```

Reply:
Yeah this makes sense, I've switched over to the originally suggested notimplemented version